addons/account/models/account_move_line.py

```diff
@@ -2965,3 +2965,16 @@ def _get_downpayment_lines(self):
         This method is overridden in the sale order module.
         '''
         return self.env['account.move.line']
+
+    # -------------------------------------------------------------------------
+    # PLACEHOLDERS
+    # -------------------------------------------------------------------------
+
+    def find_fit_product(self):
+        return ()
```

Comment:
See comment below about empty tuples

Reply:
Ideally, there should be "helper functions" for the EDIs in `account`... To find records from imported strings. We are not yet there tho.

Examples of functions doing almost the same:
Search partner in `account`
https://github.com/odoo/odoo/blob/master/addons/account/models/partner.py#L790

Search product in UBL
https://github.com/odoo/odoo/blob/b27705c901d3c228ace8dd3b86f1070c7edd9d24/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py#L701

Reply:
Also in `l10n_it_edi` this code could just extract the info from the XML, and then ask `account` generic function to search for the given product. 
![image](https://github.com/odoo/odoo/assets/1665365/93012c43-67d2-45cc-8e63-1f8b8c855607)
I'm doing a big refactor in `master` for `l10n_it_edi`, I guess we could have a secondary task to make this kind of step afterwards.